rocky/tests/katalogus/test_katalogus.py

```diff
@@ -1,51 +1,151 @@
+    mock_requests.Session().get.return_value = mock_response
+    mock_response.json.return_value = boefjes
+    member = request.getfixturevalue(member)
+
+    response = BoefjeListView.as_view()(
+        setup_request(rf.get("boefjes_list"), member.user), organization_code=member.organization.code
+    )
+    assert response.status_code == 200
+    assertContains(response, "KAT-alogus")
+    assertContains(response, "An overview of all available plugins.")
+    assertContains(response, "Boefjes")
+    assertContains(
+        response, '<li aria-current="page"><a href="/en/' + member.organization.code + '/kat-alogus/">Boefjes</a></li>'
+    )
+    assertContains(response, str(len(boefjes)) + " Boefjes available")
+    assertNotContains(response, '<span class="label-plugin-type normalizer">Normalizer</span>')
+    assertContains(response, '<span class="label-plugin-type boefje">Boefje</span>')
+    assertContains(response, "ssl-certificates")
 
 
-@pytest.mark.parametrize("member", ["superuser_member", "redteam_member"])
-def test_katalogus_plugin_listing(request, member, rf, mocker):
-    response = katalogus_plugin_listing(request, member, rf, mocker)
+@pytest.mark.parametrize("member", ["superuser_member", "admin_member", "redteam_member", "client_member"])
+def katalogus_plugin_listing_normalizers(request, rf, member, mocker):
+    normalizers = get_normalizers_data()
+    mock_requests = mocker.patch("katalogus.client.requests")
+    mock_response = mocker.MagicMock()
+    mock_requests.Session().get.return_value = mock_response
+    mock_response.json.return_value = normalizers
+    member = request.getfixturevalue(member)
 
+    response = NormalizerListView.as_view()(
+        setup_request(rf.get("normalizers_list"), member.user), organization_code=member.organization.code
+    )
     assert response.status_code == 200
+    assertContains(response, "KAT-alogus")
+    assertContains(response, "An overview of all available plugins.")
+    assertContains(response, "Normalizers")
+    assertContains(
+        response,
+        '<li aria-current="page"><a href="/en/' + member.organization.code + '/kat-alogus/">Normalizers</a></li>',
+    )
+    assertContains(response, str(len(normalizers)) + " Normalizers available")
+    assertContains(response, '<span class="label-plugin-type normalizer">Normalizer</span>')
+    assertNotContains(response, '<span class="label-plugin-type boefje">Boefje</span>')
+    assertNotContains(response, "ssl-certificates")
+    assertNotContains(response, "binaryedge")
 
-    assertNotContains(response, "You don't have permission to enable boefje")
 
-    assertContains(response, "KAT-alogus")
-    assertContains(response, "Enable")
-    assertContains(response, "BinaryEdge")
-    assertContains(response, "WPScantest")
+@pytest.mark.parametrize("member", ["superuser_member", "admin_member", "redteam_member", "client_member"])
+def katalogus_about_plugins(request, rf, member):
```

Comment:
It's a bit of an overkill to test a static template view. Checking whether the correct template is rendered would also suffice.